src/test/scala/com/databricks/spark/redshift/RedshiftSuite.scala

```diff
@@ -0,0 +1,458 @@
+/*
+ * Copyright 2015 TouchType Ltd
```

Comment:
Actually, I see now that most of this suite is based on / copied from RedshiftSourceSuite.  It would have been good to leave a comment explaining this (maybe instead of the uninformative comment that's currently at the top of this suite). It would have also been nice to factor out some of the code into common helper classes (e.g. not repeating `expectedData` in both suites).